lib/core/form/components/widgets/dynamic-table/editors/date/date.editor.ts

```diff
@@ -83,4 +83,21 @@ export class DateEditorComponent implements OnInit {
         }
     }
 
+    onFocusDateChanged(newDateValue) {
```

Comment:
Do we really need this new method? does it improve/fix anyting?

Reply:
After having a close look, I made some changes in the existing method i.e `onDateChanged`. please have a look